ttnn/cpp/ttnn/operations/reduction/generic/device/kernels/compute/reduce_h.cpp

```diff
@@ -11,28 +11,44 @@ void MAIN {
     uint32_t Ht = get_compile_time_arg_val(0);
     uint32_t Wt = get_compile_time_arg_val(1);
     uint32_t NC = get_compile_time_arg_val(2);
+    uint32_t row_chunk = get_compile_time_arg_val(3);
 
-    reduce_init<true>(tt::CBIndex::c_0, tt::CBIndex::c_2);
-    cb_wait_front(tt::CBIndex::c_2, 1);  // scaler tile from the reader
+    reduce_init<true>(tt::CB::c_in0, tt::CB::c_in2);
+    cb_wait_front(tt::CB::c_in2, 1);  // scaler tile from the reader
```

Comment:
With https://github.com/tenstorrent/tt-metal/pull/14971 we no longer have restrictions on CBs. Therefore we should move away from c_in0 etc. and use c_0 etc.

Reply:
Replaced with `tt::CBIndex::c_0` and `tt::CBIndex::c_2`